bindata/network/ovn-kubernetes/managed/ovnkube-node.yaml

```diff
@@ -56,6 +56,44 @@ spec:
       hostNetwork: true
       hostPID: true
       priorityClassName: "system-node-critical"
+      initContainers:
+      # ovnkube-node-init: wait for sbdb ready
+      {{ if eq .OVN_NODE_MODE "full" }}
+      - name: ovnkube-node-init
+        image: "{{.OvnImage}}"
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -xe
+          if [[ -f "/env/${K8S_NODE}" ]]; then
+            set -o allexport
+            source "/env/${K8S_NODE}"
+            set +o allexport
+          fi
+          echo "$(date -Iseconds) - checking sbdb"
+          ovndb_ctl_ssl_opts="-p /ovn-cert/tls.key -c /ovn-cert/tls.crt -C /ovn-ca/ca-bundle.crt"
+          transport="ssl"
+          db_port="{{.OVN_SB_PORT}}"
+          sbdb_ip="{{.OVN_SB_DB_ROUTE}}"
+          retries=0
+          while ! ovn-sbctl --no-leader-only --timeout=5 --db=${transport}:${sbdb_ip}:${db_port} ${ovndb_ctl_ssl_opts} get-connection; do
```

Comment:
`OVN_SB_DB_ROUTE` shall contain the full db address, please remove the `${transport}` and `${db_port}`, for example:

```
while ! ovn-sbctl --no-leader-only --timeout=5 --db=${sbdb_ip} ${ovndb_ctl_ssl_opts} get-connection; do
```

Please also change the `OVN_SB_DB_ROUTE` and `OVN_NB_DB_ROUTE` in ovn_kubernetes.go to:
```
data.Data["OVN_SB_DB_ROUTE"] = fmt.Sprintf("ssl:%s:%s", bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.Route, OVN_SB_ROUTE_PORT)
```

The above remove the `//` between `ssl` and Route, otherwise it would hit protocol error when connecting to sbdb route using ovn-sbctl (ovnkube cmd works fine).

Reply:
done

Reply:
@zshi-redhat am I missing an update ?
```
pkg/network/ovn_kubernetes.go:148:97: bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig undefined (type *bootstrap.OVNConfigBoostrapResult has no field or method HyperShiftConfig)
pkg/network/ovn_kubernetes.go:149:97: bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig undefined (type *bootstrap.OVNConfigBoostrapResult has no field or method HyperShiftConfig)
```

Reply:
Oh, we didn't get the route and statefulset PRs in yet, so the CI is complaining OVNKubernetesConfig.HyperShiftConfig doesn't exist. my bad, I was testing this PR on top of other unmerged PRs.

Please change to the following (current master uses env var to compose the route url):
```
	data.Data["OVN_NB_DB_ROUTE"] = fmt.Sprintf("ssl:ovnkube-sbdb-%s.apps.%s:%s", os.Getenv("HOSTED_CLUSTER_NAMESPACE"), os.Getenv("MANAGEMENT_CLUSTER_DOMAIN"), OVN_SB_ROUTE_PORT)
	data.Data["OVN_SB_DB_ROUTE"] = fmt.Sprintf("ssl:ovnkube-sbdb-%s.apps.%s:%s", os.Getenv("HOSTED_CLUSTER_NAMESPACE"), os.Getenv("MANAGEMENT_CLUSTER_DOMAIN"), OVN_SB_ROUTE_PORT)
```